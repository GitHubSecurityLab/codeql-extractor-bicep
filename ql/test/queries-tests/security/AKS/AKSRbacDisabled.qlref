security/CWE-284/AKSRbacDisabled.ql
