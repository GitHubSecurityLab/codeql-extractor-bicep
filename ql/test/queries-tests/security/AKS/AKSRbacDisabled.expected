| app.bicep:25:1:41:1 | ManagedContainerResource | AKS cluster has RBAC disabled, which can lead to unauthorized access to the cluster. |
